packages/orbit-design-tokens/.gitignore

```diff
@@ -0,0 +1,5 @@
+/node_modules
+.idea
+es
+lib
+.vscode
```

Comment:
You can remove `.node_modules`, `.vscode` and `.idea`, they are already covered by our root `.gitignore`.